apps/publish-pipeline-result/src/publishResult.ts

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env node
+
+import {
+  CompletedEvent,
+  InProgressEvent,
+  PipelineEvent,
+  PipelineResult,
+  PipelineStatus
+} from "swagger-validation-common/lib/event";
+import * as yargs from "yargs";
+import { AzureBlobClient } from "./AzureBlobClient";
+import { EventHubProducer } from "./EventHubClient";
+
+type BuildProperties = {
+  buildId: string;
+};
+
+function getBuildProperties(): BuildProperties {
+  return {
+    buildId: process.env.BUILD_BUILD_ID
+  } as BuildProperties;
+}
+
+async function getEventHubProduer(): Promise<EventHubProducer> {
+  const connectionString = process.env["EVENTHUB_CONNECTION_STRING"] || "";
```

Comment:
Why enventHub connection string from env but blob sasURL from args.

Reply:
it's also from env `AZURE_BLOB_SAS_URL`